docs/content/preview/drivers-orms/rust/yb-rust-postgres.md

```diff
@@ -0,0 +1,166 @@
+    <a href="../yb-rust-postgres/" class="nav-link">
+      YSQL
+    </a>
+  </li>
+</ul>
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../yb-rust-postgres/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YugabyteDB Rust-Postgres Smart Driver
+    </a>
+  </li>
+</ul>
+
+The [YugabyteDB Rust-Postgres Smart Driver](https://github.com/yugabyte/rust-postgres) is a rust driver for [YSQL](../../../api/ysql/) built on the [PostgreSQL rust-postgres driver,](https://github.com/sfackler/rust-postgres), with additional [connection load balancing](../../smart-drivers/) features.
+
+{{< note title="YugabyteDB Managed" >}}
+
+To use smart driver load balancing features when connecting to clusters in YugabyteDB Managed, applications must be deployed in a VPC that has been peered with the cluster VPC. For applications that access the cluster from outside the VPC network, use the upstream PostgreSQL driver instead; in this case, the cluster performs the load balancing. Applications that use smart drivers from outside the VPC network fall back to the upstream driver behaviour automatically. For more information, refer to [Using smart drivers with YugabyteDB Managed](../../smart-drivers/#using-smart-drivers-with-yugabytedb-managed).
+
+{{< /note >}}
+
+## CRUD operations
+
+The following sections demonstrate how to perform common tasks required for Rust application development using the YugabyteDB Rust smart driver.
+
+To start building your application, make sure you have met the [prerequisites](../#prerequisites).
+
+### Step 1: Set up the database connection
+
+The following table describes the connection parameters required to connect, including [smart driver parameters](../../smart-drivers/) for uniform and topology load balancing.
+
+| Parameter | Description | Default |
+| :-------- | :---------- | :------ |
+| host | Host name of the YugabyteDB instance. You can also enter [multiple addresses](#use-multiple-addresses). |  |
+| port | Listen port for YSQL | 5433 |
+| database/dbname | Database name |  |
+| user | User connecting to the database |  |
+| password | User password |  |
+| `load_balance` | [Uniform load balancing](../../smart-drivers/#cluster-aware-connection-load-balancing) | Defaults to upstream driver behavior unless set to 'true' |
+| `topology_keys` | [Topology-aware load balancing](../../smart-drivers/#topology-aware-connection-load-balancing) | If `load_balance` is true, uses uniform load balancing unless set to comma-separated geo-locations in the form `cloud.region.zone`. |
+| yb_servers_refresh_interval | If load_balance is true, the interval in seconds to refresh the servers list | 300 seconds|
+| fallback_to_topology_keys_only | When set to true, if all the servers in the primary and fallback placements are down, the smart driver does not fallback to the rest of the servers in the cluster, instead to the nodes given in the connection URL. | false |
```

Comment:
```suggestion
| fallback_to_topology_keys_only | If all the servers in the primary and fallback topology key placements are down, fall back to the host(s) specified in the connection URL, instead of to nodes across the entire cluster. | false |
```